src/actions/config.js

```diff
@@ -18,23 +18,33 @@ export function fetchConfig() {
 }
 
 export function putConfig(config) {
-  return (dispatch) => {
-    let json;
-    try {
-      json = toJSON(config);
-    } catch (e) {
-      return dispatch(addNotification(getParserErrorMessage(), e.message, 'error'));
+  return (dispatch, getState) => {
+    const errors = validateConfig(config);
+    if (errors.length) {
+      return dispatch(validationError(errors));
```

Comment:
Ops. Forgot to tell you about the view test. Could you please add that too? Sorry for that 😬 

In `views/tests/configuration.spec.js`;
```javascript
  it('should render error messages', () => {
    const { errors } = setup(Object.assign({}, defaultProps, {
      errors: ['The title field is required!']
    }));
    expect(errors.node).toExist();
  });
```